install/install.py

```diff
@@ -264,4 +265,11 @@ def danger(cls, string):
     Colorizer.info("[*] Last commit hash: %s" % owtf_last_commit())
     check_sudo()
     installer_status_code = install(sys.argv[1:])
+
+    # Copying config files
+    path = os.path.join(os.path.expanduser('~'), '.owtf', 'configuration')
+    create_directory(path)
+    path_config = os.path.join(root_dir, 'configuration')
+    dir_util.copy_tree(path_config, path)
```

Comment:
@delta24 no. I checked by deleting some files. It copies the remaining files without any error.